meta-client/src/main/java/org/apache/helix/metaclient/constants/MetaClientBadVersionException.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.helix.metaclient.constants;
+
+public class MetaClientBadVersionException extends MetaClientException {
+  public MetaClientBadVersionException() {
+    super();
+  }
```

Comment:
Same for others, do we really need such empty constructor?

Reply:
Why not? `RuntimeException` has an empty constructor. We should provide one for user who may implement a metaclient wants to use it.